### PR TITLE
Aggregate tables for OTZ outcomes and cohort retention

### PR DESCRIPTION
**Pull request by cbrianbet**
Created initial aggregate reporting table for cohort retention

### CODE REVIEW

**Comment by nobert-mumo** [comment]
@cbrianbet nice! Just checking we had documented to combine the two tables. Was it intentional to split them?

**Comment by cbrianbet** [comment]
> @cbrianbet nice! Just checking we had documented to combine the two tables. Was it intentional to split them?

@nobert-mumo we actually documented OTZOutcome separate from Cohort retention 
This PR has changes for both tables ie. AggregateOTZOutcomes and AggregateCohortRetention

**Comment by nobert-mumo** [comment]
> > @cbrianbet nice! Just checking we had documented to combine the two tables. Was it intentional to split them?
> 
> @nobert-mumo we actually documented OTZOutcome separate from Cohort retention This PR has changes for both tables ie. AggregateOTZOutcomes and AggregateCohortRetention

@cbrianbet  my bad, this makes sense.

**Review by nthusi-codes — APPROVED** [review]

File: Scripts/NDWH/load_report_tables/load_aggregate_otz_outcome.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+TransitionAttritionReason as Outcome,
+COUNT(TransitionAttritionReason) as TotalOutcome
+
+FROM NDWH.dbo.FactOTZ otz
+INNER join NDWH.dbo.DimAgeGroup age on age.AgeGroupKey=otz.AgeGroupKey
+INNER join NDWH.dbo.DimFacility f on f.FacilityKey = otz.FacilityKey
+INNER JOIN NDWH.dbo.DimAgency a on a.AgencyKey = otz.AgencyKey
+INNER JOIN NDWH.dbo.DimPatient pat on pat.PatientKey = otz.PatientKey
+INNER JOIN NDWH.dbo.DimPartner p on p.PartnerKey = otz.PartnerKey
+WHERE MFLCode > 1 and TransitionAttritionReason is not null
```
Comment (nobert-mumo) [inline Scripts/NDWH/load_report_tables/load_aggregate_otz_outcome.sql:26]: @cbrianbet I think this is already taken care on the ODS tables if am not mistaken?

**Review by nobert-mumo — APPROVED** [review]

File: Scripts/NDWH/load_report_tables/load_aggregate_otz_outcome.sql (hunk 1 above)
Comment (cbrianbet) [reply Scripts/NDWH/load_report_tables/load_aggregate_otz_outcome.sql]: Noted 
Now resolved

**Pull request merged**